### DISCUSSION TITLE
Any ETA for the next `net7-SR`?

### DISCUSSION THREAD

**Discussion by AndreasReitberger** [dotnet/maui#15694] [General]
Is there any ETA for the upcoming service release?
Thanks :)

**Comment by UkeHa** [comment]
Looking at their sprint [here](https://github.com/orgs/dotnet/projects/194/views/16) not much is being done right now. Their recent cadence was roughly monthly but who knows. Idk why they even have sprints if they don't do their releases accordingly.

**Comment by hartez** [comment]
Next service release is slated for next week.

**Reply by UkeHa** [reply]
It would be nice to be more open about the release cadence and bind it to sprints to it's easier to decide whether or not it makes sense to develop features that rely on features.

**Reply by UkeHa** [reply]
@hartez, in the same vein as last month: It would be really nice when the next releases are (either by a standard cadence or the open sprint cycles). Right now i'm waiting on the fix for collectionviews that has been backported to .net7. I know i could just use the artifacts of the nightlies but it's way easier if it's distributed to all devs through visual studio. net8 got a new preview today, will the net 7-SR be soon as well?

**Reply by hartez** [reply]
As far as I know, the next service release for 7 won't be happening until the middle of next month.

**Reply by baaaaif** [reply]
@rmarinho is there an 7.0.96 ETA?

**Reply by FM1973** [reply]
I bet the next .net 7 SR will not be available prior the "final" .net 8 release.
